src/core/core.datasetController.js

```diff
@@ -268,6 +268,49 @@ helpers.extend(DatasetController.prototype, {
 		}
 	},
 
+	/*
+	 * Returns a set of predefined style properties that should be used to represent the dataset
+	 * or the data if the index is specified
+	 * @params {number} index - data index
+	 * @return {IStyleInterface} style object
+	 */
+	getStyle: function(index) {
```

Comment:
I like the idea of creating a function here for this. Perhaps eventually the tooltip could use this too